src/main.rs

```diff
@@ -194,9 +195,21 @@ pub struct NewOptions {
     pub template: new::Template,
 }
 
+// arg_enum! {
+//     #[derive(Debug)]
+//     pub enum Target {
+//         Erlang,
+//         JavaScript
+//     }
+// }
+// https://stackoverflow.com/questions/54687403/how-can-i-use-enums-in-structopt
```

Comment:
I could not work out what Rust wanted from me here. 
Does it make sense to specify an enum for the target or we could just pass around strings as currently implemented?

Reply:
I think an enum is clean :) If we want a "js" option, we're probably better off implementing `FromStr` manually than using `arg_enum!`

Reply:
Here's an existing example of how we do this with strum https://github.com/gleam-lang/gleam/blob/main/src/new.rs#L16-L23